root/RawLogsScraper.py

```diff
@@ -166,7 +166,7 @@ def scrape(url, serverName, fileBuffer, newestAllowed = 0):
         filename = entry["name"]
         newUrl = f"{url}/{filename}"
         if entry["type"] == "directory":
-            errorCode = scrape(newUrl, serverName, fileBuffer, newestAllowed)
+            errorCode = scrape(newUrl, serverName, fileBuffer, 0)
```

Comment:
urlAge() runtimed here since it tries to get the round id from the last part of the url but the url entry was bad so it wasnt always valid

Reply:
newestAllowed needs to get passed down the chain, otherwise attempts at reconstruction will fail. I think urlAge should be ok now?